rubocop/rubocop_rails.yml

```diff
@@ -3,11 +3,34 @@ inherit_from: rubocop.yml
 AllCops:
   Exclude:
     - db/schema.rb
+    - 'bin/**/*'
+    - 'config/**/*'
     - 'vendor/**/*'
 
 Rails:
   Enabled: true
 
+Metrics/BlockLength:
+  Exclude:
+    - 'db/seeds.rb'
+    - 'spec/**/*'
+
+Metrics/LineLength:
+  Exclude:
+    - 'db/seeds.rb'
+
 Metrics/MethodLength:
   Exclude:
     - 'db/migrate/**/*'
+
+Metrics/ModuleLength:
+  Exclude:
+    - 'db/seeds.rb'
+    - 'spec/**/*'
+
+Style/StringLiterals:
+  Exclude:
+    - 'config.ru'
+    - 'Gemfile'
+    - 'Rakefile'
```

Comment:
Specific for config.ru and Rakefile, is the concern just to get a clean rubocop run with no warnings with generated code? If so, the autoformatter could clean that up for you quickly, and specifically for those 2 files they're unlikely to churn/be seen much over the life of the project.